src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportJobRecord.cs

```diff
@@ -25,7 +27,8 @@ public ExportJobRecord(Uri exportRequestUri)
             SchemaVersion = 1;
             Status = OperationStatus.Queued;
             Id = Guid.NewGuid().ToString();
-            QueuedTime = DateTimeOffset.UtcNow;
+            SecretName = SecretPrefix + Id;
```

Comment:
We should think about this a bit more. Currently, I am planing to implement the dedup logic in the stored procedure which means in the case where there is matching job, the existing job will be returned, which would contain a different id than the one generated here (I was also going to move the id generation to the stored procedure).

Reply:
You don't need to change this right now in this PR.